README.md

```diff
@@ -60,61 +60,27 @@ A [Dockerfile](Dockerfile) is provided for convenience.
-- [Facebook](https://developers.facebook.com/)
-	- Create an application
-	- Add the Facebook Login product from the left panel
-	- Enable Client OAuth Login, Web OAuth Login, and Embedded Browser OAuth Login
-	- List all testing / production callback URLs in Valid OAuth redirect URLs
-		- Should be in the format: `https://YOUR_DOMAIN/auth/facebook/callback`
-		- For local testing: `http://localhost:3000/auth/facebook/callback`
-	- Optionally, repeat the process for separate testing and production applications
-
 Environment Variable | Description
 ---------------------|------------
-PRODUCTION | Set to `true` to set OAuth callbacks to production URLs (default: `false`)
+PRODUCTION | Set to `true` to enable reverse proxy trusting (default: `false`)
 PORT | The port the check in system should run on (default: `3000`)
-MONGO_URL | The URL to the MongoDB server (default: `mongodb://localhost/`)
-UNIQUE_APP_ID | The MongoDB database name to store data in (default: `registration`)
+MONGO_URL | The URL to the MongoDB server (default: `mongodb://localhost/registration`)
 VERSION_HASH | The Git short hash used to identify the current commit (default: parsed automatically from the `.git` folder, if it exists)
-*SOURCE_VERSION* | Same as `VERSION_HASH` but overrides it if present. Used by Deis.
-*WORKFLOW_RELEASE_CREATED_AT* | Provided by Deis (default: `null`)
-*WORKFLOW_RELEASE_SUMMARY* | Provided by Deis (default: `null`)
+ADMIN_KEY_SECRET | A API key used to authenticate as admin an access the GraphQL api (default: random key that changes every server restart)
 COOKIE_MAX_AGE | The `maxAge` of cookies set in milliseconds (default: 6 months) **NOTE: this is different from the session TTL**
 COOKIE_SECURE_ONLY | Whether session cookies should sent exclusively over secure connections (default: `false`)
 PASSWORD_RESET_EXPIRATION | The time that password reset links sent via email should be valid for in milliseconds (default: 1 hour)
 SESSION_SECRET | The secret used to sign and validate session cookies (default: random 32 bytes regenerated on every start up)
-GITHUB_CLIENT_ID | OAuth client ID for GitHub *required*
-GITHUB_CLIENT_SECRET | OAuth client secret for GitHub *required*
-GOOGLE_CLIENT_ID | OAuth client ID for Google *required*
-GOOGLE_CLIENT_SECRET | OAuth client secret for Google *required*
-FACEBOOK_CLIENT_ID | OAuth client ID for Facebook *required*
-FACEBOOK_CLIENT_SECRET | OAuth client secret for Facebook *required*
+GROUND_TRUTH_URL | Base URL of [Ground Truth](https://github.com/HackGT/ground-truth) instance (e.g. `https://login.hack.gt`) *required*
+GROUND_TRUTH_ID | OAuth client ID from Ground Truth *required*
+GROUND_TRUTH_SECRET | OAuth client secret from Ground Truth *required*
 EMAIL_FROM | The `From` header for sent emails (default: `HackGT Team <hello@hackgt.com>`)
 EMAIL_KEY | The SendGrid API key for sending emails (default: *none*)
```

Comment:
if the default is none, then isn't this required?

Reply:
Yep good catch